diff --git a/node_modules/monaco-editor-textmate/dist/tm-to-monaco-token.js b/node_modules/monaco-editor-textmate/dist/tm-to-monaco-token.js
index 9f673bb..b64bef2 100755
--- a/node_modules/monaco-editor-textmate/dist/tm-to-monaco-token.js
+++ b/node_modules/monaco-editor-textmate/dist/tm-to-monaco-token.js
@@ -34,11 +34,11 @@ exports.TMToMonacoToken = (editor, scopes) => {
             const char = scope[i];
             if (char === ".") {
                 const token = scope.slice(0, i);
-                if (editor['_themeService'].getTheme()._tokenTheme._match(token + "." + scopeName)._foreground >
+                if (editor['_themeService']._theme._tokenTheme._match(token + "." + scopeName)._foreground >
                     1) {
                     return token + "." + scopeName;
                 }
-                if (editor['_themeService'].getTheme()._tokenTheme._match(token)._foreground > 1) {
+                if (editor['_themeService']._theme._tokenTheme._match(token)._foreground > 1) {
                     return token;
                 }
             }
diff --git a/node_modules/monaco-editor-textmate/src/tm-to-monaco-token.ts b/node_modules/monaco-editor-textmate/src/tm-to-monaco-token.ts
index 36eef7b..0c729dc 100755
--- a/node_modules/monaco-editor-textmate/src/tm-to-monaco-token.ts
+++ b/node_modules/monaco-editor-textmate/src/tm-to-monaco-token.ts
@@ -37,12 +37,12 @@ export const TMToMonacoToken = (editor: monacoNsps.editor.ICodeEditor, scopes: s
             if (char === ".") {
                 const token = scope.slice(0, i);
                 if (
-                    editor['_themeService'].getTheme()._tokenTheme._match(token + "." + scopeName)._foreground >
+                    editor['_themeService']._theme._tokenTheme._match(token + "." + scopeName)._foreground >
                     1
                 ) {
                     return token + "." + scopeName;
                 }
-                if (editor['_themeService'].getTheme()._tokenTheme._match(token)._foreground > 1) {
+                if (editor['_themeService']._theme._tokenTheme._match(token)._foreground > 1) {
                     return token;
                 }
             }
