diff --git a/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js b/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js
index d376d1e..6f9caca 100644
--- a/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js
+++ b/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js
@@ -6016,7 +6016,7 @@ async function fileToBuiltUrl(id, config, pluginContext, skipPublicCheck = false
     const postfix = (search || '') + (hash || '');
     const content = await fs$t.promises.readFile(file);
     let url;
-    if (config.build.lib ||
+    if ((config.build.lib && !file.endsWith('.woff') && !file.endsWith('.woff2')) ||
         (!file.endsWith('.svg') &&
             content.length < Number(config.build.assetsInlineLimit))) {
         // base64 inlined as a string
@@ -24095,7 +24095,7 @@ function buildImportAnalysisPlugin(config) {
             }
         },
         async transform(source, importer) {
-            if (importer.includes('node_modules') &&
+            if (config.build.lib || importer.includes('node_modules') &&
                 !source.includes('import.meta.glob')) {
                 return;
             }
@@ -71218,20 +71218,27 @@ function webWorkerPlugin(config) {
             }
             let url;
             if (isBuild) {
-                if (query.inline != null) {
-                    // bundle the file as entry to support imports and inline as blob
-                    // data url
-                    const rollup = require('rollup');
-                    const bundle = await rollup.rollup({
-                        input: cleanUrl(id),
-                        plugins: config.plugins
+                // bundle the file as entry to support imports
+                const rollup = require('rollup');
+                const bundle = await rollup.rollup({
+                    input: cleanUrl(id),
+                    plugins: await resolvePlugins({ ...config }, [], [], [])
+                });
+                let code;
+                try {
+                    const { output } = await bundle.generate({
+                        format: 'iife',
+                        sourcemap: config.build.sourcemap
                     });
-                    try {
-                        const { output } = await bundle.generate({
-                            format: 'es',
-                            sourcemap: config.build.sourcemap
-                        });
-                        return `const blob = new Blob([atob(\"${Buffer.from(output[0].code).toString('base64')}\")], { type: 'text/javascript;charset=utf-8' });
+                    code = output[0].code;
+                }
+                finally {
+                    await bundle.close();
+                }
+                const content = Buffer.from(code);
+                if (query.inline != null) {
+                    // inline as blob data url
+                    return `const blob = new Blob([atob("${content.toString('base64')}")], { type: 'text/javascript;charset=utf-8' });
             export default function WorkerWrapper() {
               const objURL = (window.URL || window.webkitURL).createObjectURL(blob);
               try {
@@ -71240,16 +71247,14 @@ function webWorkerPlugin(config) {
                 (window.URL || window.webkitURL).revokeObjectURL(objURL);
               }
             }`;
-                    }
-                    finally {
-                        await bundle.close();
-                    }
                 }
                 else {
-                    // emit as separate chunk
+                    const basename = path__default.parse(cleanUrl(id)).name;
+                    const contentHash = getAssetHash(content);
                     url = `__VITE_ASSET__${this.emitFile({
-                        type: 'chunk',
-                        id: cleanUrl(id)
+                        name: `${basename}.${contentHash}.js`,
+                        type: 'asset',
+                        source: code
                     })}__`;
                 }
             }
